src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -1630,6 +1649,9 @@ private function apply_attributes_updates( $shift_this_point = 0 ) {
 		 * replacements adjust offsets in the input document.
 		 */
 		foreach ( $this->bookmarks as $bookmark_name => $bookmark ) {
+			$bookmark_start = $bookmark->start;
+			$bookmark_end   = $bookmark_start + $bookmark->length;
```

Comment:
Could these allocations be removed?

Reply:
`$bookmark_end` is hard to remove because we have to compute it, and I left `$bookmark_start` in for symmetry. it wasn't offering much otherwise so I took it out in 449ae5ad0e

Reply:
these aren't really allocations either, as they will be created locally on the stack.